src/Domain/Entities/Servers/GameServerConfig.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Crpg.Domain.Common;
+using Crpg.Domain.Entities.Servers.Enums;
+
+namespace Crpg.Domain.Entities.Servers;
+public class GameServerConfig : AuditableEntity
+{
+    public ServerIdentifier Identifier { get; set; }
+    public string? ServerName { get; set; }
+    public string? GameType { get; set; }
+    public string? GamePassword { get; set; }
+    public string? WelcomeMessage { get; set; }
+    public List<string> Maps { get; set; } = new List<string>();
+    public bool? AllowPollsToKickPlayers { get; set; }
+    public bool? AllowPollsToChangeMaps { get; set; }
+    public string? DisableCultureVoting { get; set; }
+}
```

Comment:
You need to check every ds_config txt file so you make sure you have every single parameters. You're missing a lot
The class i gave you where only the begining , they need to be exhaustive